samples/tizen-bridge-native/tizen_bridge_native.c

```diff
@@ -14,7 +14,7 @@
  */
 
 #include "iotjs.h"
-#include "iotjs_module_tizen.h"
```

Comment:
This sample assumes that an `include path` containing `iotjs_tizen_service_app.h` is set in compile time. So we can use `iotjs_tizen_service_app.h`.

Reply:
I fixed it.